neil

```diff
@@ -52,14 +52,17 @@
 
 (defn url-encode [s] (URLEncoder/encode s "UTF-8"))
 
-(def github-user (or (System/getenv "NEIL_GITHUB_USER")
-                     (System/getenv "BABASHKA_NEIL_DEV_GITHUB_USER")))
-(def github-token (or (System/getenv "NEIL_GITHUB_TOKEN")
-                      (System/getenv "BABASHKA_NEIL_DEV_GITHUB_TOKEN")))
+(def github-user-envvars ["NEIL_GITHUB_USER" "BABASHKA_NEIL_DEV_GITHUB_USER"])
+(def github-token-envvars ["NEIL_GITHUB_TOKEN" "BABASHKA_NEIL_DEV_GITHUB_TOKEN"])
+
+(def github-user (some #(System/getenv %) github-user-envvars))
+(def github-token (some #(System/getenv %) github-token-envvars))
```

Comment:
This change didn't seem to be necessary?

Reply:
Ah I see you're using the env var names in the following, never mind

Reply:
This change (and some others in the `neil` script) are consequences of gen'ing the script - that is to say, the change that caused this change is actually already on main (these lines in particular come from #238 ), and the generated `neil` script on main is out of sync with the sources from which it is generated.

If preferred, I can roll back the changes to the generated script (i.e. leave it out of sync) , but they'll presumably get re-added to the final product when the script is genned. Or maybe I'm missing something about how the output artifact gets updated...

Reply:
ok, no this is fine